#155124480 Heroku deployment

shakirandagire opened the pull request:
#### What does this PR do?
Heroku deployment

#### Description of Task to be completed?
Ensure the project is deployed on heroku for staging.

#### How should this be manually tested?
https://wg-logros.herokuapp.com

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[155124480](https://www.pivotaltracker.com/story/show/155124480)

muchai-mercy reviewed (changes requested):
@shakirandagire Did you make 1068 files changes. Not sure where to start reviewing

shakirandagire commented:
Let me work on the gitignore and push to reduce the number of files changed

shakirandagire commented:
I have worked on it and now the files have reduced to 9 changed files

muchai-mercy commented on wger/settings_global.py:
```diff
@@ -317,13 +319,14 @@
 SOCIAL_AUTH_FACEBOOK_SECRET = 'a4fcf058e205ee880313053339ce3924'
 
 
+# STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
```
Should this be commented out? If not in use, we can clear it out.

muchai-mercy commented on .gitignore:
```diff
@@ -65,3 +64,5 @@ node_modules
 .DS_Store
 venv-django
 .vscode/
+# .env
```
Do we need this? @shakirandagire given you already added `.env`?

shakirandagire replied:
@Mercy-Muchai I had put it as a title. But let me remove it since its not necessary

shakirandagire replied:
@Mercy-Muchai I removed the # .env from gitignore file

kevinsamoei reviewed (approved):
LGTM

JoyyToo reviewed (approved):
LGTM

kevinsamoei commented:
@Mercy-Muchai The settings file has been updated. The unused code has been removed. Review the changes.

kevinsamoei committed:
[Chore #155124480] Heroku deployment:
- Add procfile
- Add whitenoise to settings
- Add dj_database_url to settings
- Modify site_url
- Modify static_root
- Add runtime.txt
- Change package.json
- Add bower.json
- Modify bower installed apps

[Delivers #155124480]

muchai-mercy reviewed (approved):
LGTM
